### ISSUE TITLE
Error in upgrading npm from 3.0.0 to 3.4.1

### ISSUE THREAD

**Issue by mohitr-optimus** [felixrieseberg/npm-windows-upgrade#56]
Upgrading (fallback method)... /You wanted to install npm 3.4.1, but the install
ed version is3.0.0.
Please consider reporting your trouble to http://aka.ms/npm-issues.

Debug Information:

{ http_parser: '2.5.0',
  node: '4.2.4',
  v8: '4.5.103.35',
  uv: '1.7.5',
  zlib: '1.2.8',
  ares: '1.10.1-DEV',
  icu: '56.1',
  modules: '46',
  openssl: '1.0.2e',
  npm: '3.0.0',
  os: 'win32 x64' }

**Comment by felixrieseberg** [comment]
Thanks for reporting! Quick question: Do you remember how you installed Node?

**Comment by mohitr-optimus** [comment]
@felixrieseberg : Sorry for the delay in response. I have installed node by windows msi provided by node (https://nodejs.org/en/download/). After that I have downgraded the npm to 3.0.0 by npm-windows-upgrade, now whenever I tried to install any module using the npm install command it freezes in few times and nothing is processed further. Therefore I have again tried to upgrade it to its original version and the above error is displayed in the cmd window.
